components/fieldgroups/healthcare-plan/healthcare-plan-details.schema.json

```diff
@@ -0,0 +1,92 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/healthcare-plan-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Healthcare Plan Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["healthcare_services"]
+  },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/classes/healthcare-plan"],
+  "description": "Details of a Health Insurance plan provided by an organization.",
+  "definitions": {
+    "plan-details": {
+      "properties": {
+        "xdm:payerID": {
+          "Title": "Payer ID",
+          "type": "string",
+          "description": "Unique identifier of the payer."
+        },
+        "xdm:contractID": {
```

Comment:
Contract looks more like an agreement between Plan and a member so, it should be either removed from here or moved to the plans array in member if required.